lib/serverMethods.js

```diff
@@ -0,0 +1,34 @@
+const { collection } = require("./database");
+
+async function getCollection(collectionName) {
+  const promises = await collection(collectionName).find({}).toArray();
+  const data = await promises;
+  return data;
```

Comment:
I think `promises` is already your `data`.

```suggestion
  return await collection(collectionName).find({}).toArray();
```

Reply:
Thanks for the feedback. Added the change into the project 🚀